align_system/algorithms/oracle_adm.py

```diff
@@ -17,30 +15,15 @@
 
 
 class OracleADM(ActionBasedADM):
-    def __init__(
-        self, probabilistic: bool=False, upweight_missing_kdmas: bool=False,
-        filter_unlabeled_actions: bool=False, misaligned: bool=False, **kwargs
-    ):
-        self.probabilistic = probabilistic
-        self.upweight_missing_kdmas = upweight_missing_kdmas
-        self.filter_unlabeled_actions = filter_unlabeled_actions
+    def __init__(self, misaligned: bool=False, **kwargs):
         self.misaligned = misaligned
 
     def choose_action(self, scenario_state, available_actions, alignment_target,
-                      distribution_matching: str='sample',
-                      kde_norm: str='rawscores', **kwargs):
+                      distribution_matching='sample', kde_norm='rawscores',
+                      probabilistic=False **kwargs):
```

Comment:
Missing a comma after probabilistic

Reply:
Fixed in [7f146b3](https://github.com/ITM-Kitware/align-system/pull/106/commits/7f146b39609076b556f247f42c774ce6863d2269)